nakamoto/roster.json

```diff
@@ -49,6 +49,10 @@
       "materials": [
         {
           "type": "cv"
+        },
+        {
+          "type": "video",
+          "url" : "https://youtu.be/qDuddSiA3uU"
         }
```

Comment:
Can you please change `"url" : "https://youtu.be/qDuddSiA3uU"` to `"url" : "qDuddSiA3uU"` :) ?